charts/opensearch/Chart.yaml

```diff
@@ -30,3 +30,5 @@ maintainers:
   - name: peterzhuamazon
   - name: prudhvigodithi
   - name: TheAlgo
+  - name: phijojo
```

Comment:
We wont be able to add you as a maintainer without enough nominations. Please refer to at [maintainers.md](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer) guide to understand the process to become a maintainer.

Reply:
Good to know, thanks for sharing the document. I  have removed my name from the maintainer's list.